src/docfx/build/toc/parser/TableOfContentsParser.cs

```diff
@@ -205,15 +205,15 @@ private static List<string> GetMonikers(Context context, TableOfContentsItem cur
             {
                 foreach (var item in currentItem.Items)
                 {
-                    if (item.Monikers?.Count == 0)
+                    if (!item.Monikers.Any())
```

Comment:
https://stackoverflow.com/questions/305092/which-method-performs-better-any-vs-count-0
For `List`, it is better to use `Count` since it is a prepared property. For `IEnumerable`, it would be better to use `Any()` other than `Count()`.